Assignment2 crud operation

```diff
@@ -0,0 +1,164 @@
+//Kalpvriksha Program | Batch-2025: C Programming Assignments covering CRUD operations.
+
+//assginment 2: CRUD operation, by creating a file named users.txt
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+typedef struct
+{
+    int id;
+    char name[50];
+    int age;
+} User;
+
+void addUser();
+void displayUsers();
+void updateUser();
+void deleteUser();
+
+int main()
+{
+    int choice;
+    do
+    {
+        printf("1. Add User\n2. Display Users\n3. Update User\n4. Delete User\n5. Exit\n");
+        printf("Enter your choice: ");
+        scanf("%d", &choice);
+
+        switch (choice)
+        {
+        case 1:
+            addUser();
+            break;
+        case 2:
+            displayUsers();
+            break;
+        case 3:
+            updateUser();
+            break;
+        case 4:
+            deleteUser();
+            break;
+        case 5:
+            printf("Exiting...\n");
+            break;
+        default:
+            printf("Invalid choice!\n");
+        }
+    } while (choice != 5);
+
+    return 0;
+}
+
+void addUser()
+{
+    FILE *file = fopen("users.txt", "a");
+    User user;
+
+    printf("Enter ID: ");
+    scanf("%d", &user.id);
+    printf("Enter Name: ");
+    scanf("%s", user.name);
```

Comment:
can we handle read input and printing output in separate function? take care at all places.